### ISSUE TITLE
Multi-sig Integration

### ISSUE THREAD

**Issue by wgmi-rambo** [wgmi-rambo/nft-contract#1]
Right now it goes to the contract owner address:

```sol
function withdraw() public payable onlyOwner {
    (bool success, ) = payable(owner()).call{value: address(this).balance}("");
    require(success, "withdrawal failed");
}
```

**Comment by wgmi-rambo** [comment]
We need to integrate a multi-sig wallet somehow.

Possibilities:

1. Multi-sig wallet is the owner of the contract.
  a. This means that simple things like setting the whitelist would need multiple signatures. Not sure about the gas-cost implications of this.
2. A trusted community member is the owner, and withdrawals can only be made to a pre-specified address (the multi-sig address)
  a. Need to create a getter method for address state variable.
  b. Will the address be settable? By whom?

**Comment by wgmi-rambo** [comment]
We're discussing this issue on Discord, here:
#dev-talk/multi-sig: https://discord.com/channels/908727219600388207/912380991786590269

**Comment by wgmi-rambo** [comment]
Closed by #5
